eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/steps/processors/ExcludeStructuralVariantsProcessor.java

```diff
@@ -0,0 +1,100 @@
+
+    private static String basesRegEx = "([ACTGNactgn]+)";
+
+    private static String altIDRegEx_positive_match = "([\\p{Alnum}|[\\p{Punct}&&[^,<>]]]+)";
+
+    private static String altIDRegEx_negative_match = "([\\p{Punct}]+)";
+
+    private static String altIDRegEx = String.format("((?!%s)%s)", altIDRegEx_negative_match,
+                                                     altIDRegEx_positive_match);
+
+    private static String stdPrefixRegEx = MessageFormat.format(
+            "<DEL>|<INS>|<DUP>|<INV>|<CNV>|<DUP:TANDEM>|<DEL:ME:{0}>|<INS:ME:{0}>", "(\\p{Alnum})+");
+
+    private static String altIndelRegEx = String.format("(%s|\\*)", stdPrefixRegEx);
+
+    private static String altOtherRegEx = String.format("((?!%s)%s)", stdPrefixRegEx,
+                                                        String.format("<%s>", altIDRegEx));
+
+    /**
+     * See <a href="https://github.com/EBIvariation/vcf-validator/blob/be6cf8e2b35f2260166c1e6ffa1258a985a99ba3/src/vcf/vcf_v43.ragel#L190">VCF specification grammar</a>
+     */
+    private static String altSVRegEx = String.join("|",
+                                                   String.format("(\\]%s:%s\\]%s)", chromosomeRegEx, positionRegEx,
+                                                                 basesRegEx),
+                                                   String.format("(\\[%s:%s\\[%s)", chromosomeRegEx, positionRegEx,
+                                                                 basesRegEx),
+                                                   String.format("(%s\\]%s:%s\\])", basesRegEx, chromosomeRegEx,
+                                                                 positionRegEx),
+                                                   String.format("(%s\\[%s:%s\\[)", basesRegEx, chromosomeRegEx,
+                                                                 positionRegEx),
+                                                   String.format("(\\.%s)", basesRegEx),
+                                                   String.format("(%s\\.)", basesRegEx));
+
+    private static String altGVCFRegEx = "(<\\*>)";
+
+    /**
+     * See <a href="https://github.com/EBIvariation/vcf-validator/blob/be6cf8e2b35f2260166c1e6ffa1258a985a99ba3/src/vcf/vcf_v43.ragel#L201">VCF specification grammar</a>
+     */
+    private static String STRUCTURAL_VARIANT_REGEX = String.format("^(%s|%s|%s|%s)$", altIndelRegEx, altSVRegEx,
+                                                                   altGVCFRegEx, altOtherRegEx);
+
+    private static final Pattern STRUCTURAL_VARIANT_PATTERN = Pattern.compile(STRUCTURAL_VARIANT_REGEX);
+
+    @Override
+    public IVariant process(IVariant variant) {
+        Matcher matcher = STRUCTURAL_VARIANT_PATTERN.matcher(variant.getAlternate());
+        if(matcher.matches()) {
+            logger.info("Skipped processing variant " + variant.toString() + " because it was a structural variant.");

```

Comment:
Given the `toString` method is going to be quite long, maybe this message could be:

> "Skipped processing structural variant " + variant.toString()